firmware/projects/EV5/debug/FrontControllerSimple/config.yaml

```diff
@@ -2,8 +2,6 @@ canGen:
   ourNode: fc
   busses:
     - busName: veh
-      dbcFiles:
-        - "../../veh.dbc"
+      dbcFile: "../../veh.dbc"
```

Comment:
@BlakeFreer wouldn't it be better to give paths starting from racecar instead of using relative paths. 

For example this would become
```yaml
    dbcFile: "firmware/projects/EV5/veh.dbc"
```

I understand for simple projects this becomes verbose, but for projects that start to have more nested folder structures like this one, it seems pretty messy to have it this way.

Reply:
Projects should be nested according to their logical grouping with other projects. CAN buses are defined for such a group of projects, so the nesting will not become too deep.

Using an absolute path like `firmware/.../veh.dbc` couples the config.yaml to the folder structure beyond the project group. If I rename or move the folder holding a group of projects and associated DBCs, then the config.yaml will need to be updated if it uses absolute paths, whereas it wouldn't using these relative paths.

Also `firmware/projects/EV5/veh.dbc` is much  more verbose than `../../veh.dbc`